kbfsmd/root_metadata_v2_test.go

```diff
@@ -52,6 +52,20 @@ func TestRootMetadataVersionV2(t *testing.T) {
 	require.Equal(t, PreExtraMetadataVer, rmd.Version())
 }
 
+func TestWriterMetadataV2Empty(t *testing.T) {
```

Comment:
Also found that this is covered by `testWriterMetadataV2UnchangedEncoding`, but doesn't hurt to have a more specific test.